dap_prinz_green_jobs/analysis/ojo_analysis/process_ojo_green_measures.py

```diff
@@ -0,0 +1,753 @@
+"""
+Load the green measures from a sample of OJO data and process it into a form needed to create analysis from
+"""
+
+from dap_prinz_green_jobs import BUCKET_NAME, logger
+from dap_prinz_green_jobs.getters.data_getters import load_s3_data
+from dap_prinz_green_jobs.getters.industry_getters import load_sic
+from dap_prinz_green_jobs.getters.ojo_getters import (
+    get_mixed_ojo_location_sample,
+    get_mixed_ojo_salaries_sample,
+)
+from dap_prinz_green_jobs.pipeline.green_measures.occupations.occupations_measures_utils import (
+    OccupationMeasures,
+)
+
+import pandas as pd
+import numpy as np
+import altair as alt
+from tqdm import tqdm
+
+import os
+
+
+# clean up skills
+def merge_ents(ents):
+    if not isinstance(ents, list):
+        return None
+
+    elif "green" in ents[1]:
+        return [ents[0]] + [ents[1][0]] + [ents[1][1]] + ents[1][2]
+    else:
+        return ents[0] + [ents[1]]
+
+
+# function to clean SOC names
+def clean_soc_name(soc_name):
+    if soc_name:
+        return soc_name.replace("n.e.c.", "").strip()
+    else:
+        return None
+
+
+def load_ojo_green_measures(
+    production, config, skills_date_stamp, occ_date_stamp, ind_date_stamp
+):
```

Comment:
doc strings + typing hints - should we add this in the ojo getter instead? up for debate!